### PR TITLE
 change classes to BEM

### PR DESCRIPTION
**Pull request by Zaid0307**
change the Classes to Bam

### CODE REVIEW

**Comment by Zaid0307** [comment]
Add card.css and change the classes, did not import the new fil yet. more changes will follow

File: Styles/Card.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*.content__box__wgbox,
+  .content__box__conduentbox,
+  .content__box__kmabox */
```
Comment (macfrei) [inline Styles/Card.css:3]: Do you need the comments here?

File: Styles/Card.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+.card {
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  width: 100%;
+}
+
+/*.content__box__wgbox ul,
+  .content__box__conduentbox ul,
+  .content__box__kmabox ul */
+.card ul {
```
Comment (macfrei) [inline Styles/Card.css:13]: To be more specific, I would add a class here instead: 
```suggestion
.card__list {
```

File: Styles/styles.css [hunk 3]
```diff
@@ -20,23 +20,23 @@ main {
   align-items: center;
```
Comment (macfrei) [inline Styles/styles.css:20]: I would recommend restructuring your styles in different CSS files :)

**Review by macfrei — COMMENTED** [review]
Nice job, Zaid :)

File: Styles/styles.css (hunk 3 above)
Comment (Zaid0307) [reply Styles/styles.css:20]: Thanks for the comments. 
for now i like to keep the comments to have an better overview till i have an idea for the different css files.
I will also change .card ul in to card__list.

**Pull request merged**